src/src/EventSubscriber/MiddlewareSubscriber.php

```diff
@@ -31,24 +29,26 @@ public function onKernelController(ControllerEvent $event): void
         $reflectionMethod = new \ReflectionMethod($controller[0], $controller[1]);
         $attributes = $reflectionMethod->getAttributes();
 
-        $run_middleware_instance= null;
+        $middleware = null;
 
         foreach ($attributes as $attribute) {
-
-            if ($attribute->getName() !== RunMiddleware::class) {
+            if (RunMiddleware::class !== $attribute->getName()) {
                 continue;
             }
 
-            if ($run_middleware_instance == null) {
-                $run_middleware_instance = $attribute->newInstance();
-                $middleware = $run_middleware_instance;
+            if (null == $middleware) {
+                $middleware = $attribute->newInstance();
             }
-            
+
+            if (null === $middleware) {
+                continue;
+            }
+
             $firstMethodName = $middleware->name;
             $otherMethodNames = $middleware->getExtras();
-            
+
             $methodNames = [$firstMethodName, ...$otherMethodNames];
-            
+
             $middleware_result = null;
```

Comment:
Will test these changes. Looks like just gets rid of the `$run_middleware_instance` variable, but still ends up doing the same operations using only the `$middleware` var.